lib/global_session/session/v4.rb

```diff
@@ -61,6 +62,7 @@ def to_s
         payload[EXPIRED_AT] = @expired_at.to_i
         payload[ISSUED_AT] = @created_at.to_i
         payload[ISSUER] = @directory.local_authority_name
+        payload[SCOPE] = @scope
```

Comment:
I'm not sure that this gem needs to concern itself with the `scp` claim. Consider that it doesn't concern itself with the `sub` claim, for example (or the `user`/`instance` claims of legacy sessions) .. to the global session gem, these are all simply key/value pairs in the payload. There is a `Configuration` element that tells the gem which keys are allowed to exist in the payload (so it knows which should be secure and which should be insecure), but the intention was to let different applications configure their sessions with different contents.

For ID, expired_at, issued_at and similar claims, the gem needs to care about them because it makes decisions based on them ... these are the exception, however. Since we call `@signed.dup` the scope should get copied into the `to_s` representation just as `sub` would.

Seem reasonable to you, if we have the gem stop caring about `scp` and `SCOPE` and such?